homework-classes/Contributor.js

```diff
@@ -14,6 +13,21 @@ class Contributor {
    */
   render(container) {
     // TODO: replace the next line with your code.
-    Util.createAndAppend('pre', container, { text: JSON.stringify(this.contributor, null, 2) });
+    const contributorsDiv = Util.createAndAppend('div', root, { id: 'contributors' });
+    for (let contributor of listOfContributors) {
+      Util.createAndAppend('img', contributors, {
+        src: this.contributor.avatar_url,
+        class: 'contributors-detail',
+      });
+      Util.createAndAppend('div', contributors, {
+        text: this.contributor.login,
+        class: 'contributors-detail',
+      });
+      Util.createAndAppend('div', contributors, {
+        text: this.contributor.contributions,
+        class: 'contributors-detail',
+      });
+    }
+    //Util.createAndAppend('pre', container, { text: JSON.stringify(this.contributor, null, 2) });
```

Comment:
You'll then want to do a normal append with what you created above. Something like:
```
container.appendChild(contributorsDiv)
```